File: java/different.java
import java.util.Scanner;

public class different {
	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);
		while (scan.hasNextLong())
			System.out.println(Math.abs(scan.nextLong() - scan.nextLong()));
		scan.close();
	}
}

File: ts/src/different.ts
import readline = require('readline')

readline
  .createInterface({
    input: process.stdin,
    output: process.stdout
  })
  .on('line', (line: string) => {
    let [a, b] = line.split(' ').map(Number)
    a ??= 0
    b ??= 0

    console.log(Math.abs(a - b))
  })
  .on('close', () => {
    process.exit(0)
  })
